terra_notebook_utils/drs.py

```diff
@@ -290,6 +294,34 @@ def copy_to_bucket(drs_uri: str,
         dst_url += f"/{dst_key}"
     copy(drs_uri, dst_url, indicator_type, workspace_name, workspace_namespace)
 
+def copy_batch(manifest: List[Dict[str, str]] = None,
+               drs_urls: Iterable[str] = None,
+               dst_pfx: str = None,
+               indicator_type: Indicator = Indicator.notebook_bar if is_notebook() else Indicator.log,
+               workspace_name: Optional[str] = WORKSPACE_NAME,
+               workspace_namespace: Optional[str] = WORKSPACE_NAMESPACE):
+    assert (manifest is None) != (drs_urls is None), "Specify either 'manifest' or 'drs_urls' and 'dst_pfx'"
+    if manifest:
+        assert dst_pfx is None
+        copy_batch_manifest(manifest, indicator_type, workspace_name, workspace_namespace)
+    elif drs_urls:
+        assert dst_pfx is not None, ''
+        copy_batch_urls(drs_urls, dst_pfx, indicator_type, workspace_name, workspace_namespace)
+    else:
+        assert False
```

Comment:
Could this be upgraded to a proper error with a message?

Reply:
I changed the other asserts to be more explicit exceptions to make the distinction clear, but this assert will never fire. `assert False` is just a helpful message to the programmer that this code path (under normal operations) should never be reached.

Reply:
This could be reached if the user specifies either `drs.copy_batch(manifest=[])` or `drs.copy_batch(drs_urls=[])`.  In which case, what would be the best behavior?  A message to the user that nothing was specified and we're exiting?  I don't think it should be a value error or an `assert False` either.